src/main/java/frc/robot/subsystems/BallTransport.java

```diff
@@ -29,6 +29,8 @@
     static private final double TRANSPORT_SPEED = 1.0;
 
     boolean seenBall = false;
+    boolean bottomFlag = false;
+    boolean topFlag = false;
```

Comment:
What does `flag` mean here? That it saw a ball? `sawTopBall` might be a better name here.